src/handlers/watch-user-activity.ts

```diff
@@ -1,27 +1,28 @@
 import { getWatchedRepos } from "../helpers/get-watched-repos";
 import { updateTaskReminder } from "../helpers/task-update";
-import { Context } from "../types/context";
 import { ListForOrg, ListIssueForRepo } from "../types/github-types";
+import { ContextPlugin } from "../types/plugin-input";
 
-export async function watchUserActivity(context: Context) {
+export async function watchUserActivity(context: ContextPlugin) {
   const { logger } = context;
 
   const repos = await getWatchedRepos(context);
 
   if (!repos?.length) {
-    logger.info("No watched repos have been found, no work to do.");
-    return false;
+    return { message: logger.info("No watched repos have been found, no work to do.").logMessage.raw };
   }
 
-  for (const repo of repos) {
-    logger.debug(`> Watching user activity for repo: ${repo.name} (${repo.html_url})`);
-    await updateReminders(context, repo);
-  }
+  await Promise.all(
+    repos.map(async (repo) => {
+      logger.debug(`> Watching user activity for repo: ${repo.name} (${repo.html_url})`);
+      await updateReminders(context, repo);
+    })
+  );
 
-  return true;
+  return { message: "OK" };
```

Comment:
How about a statusCode: 200 instead

Reply:
Sure, will change it. Thought it was not useful because the network call already sends a `200`, this is the body.